src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
```

Comment:
This class should be abstract
```suggestion
public abstract class Figure implements Drawable, AreaCalculator {
```